src/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -0,0 +1,13 @@
+type BackgroundImageProps = {
```

Comment:
You can add _as_ prop to make component more adaptable in dynamic layouts or accessibility needs

Reply:
As @NacomiTagiera  wrote. We need to code this component so that the src to the public image is the props we will pass as the background image. This way we will use your component many times in the project.